src/views/List.jsx

```diff
@@ -10,46 +11,113 @@ export function List({ data, listToken }) {
 		setSearchQuery('');
 	};
 
+	//data is getting passed through firebase.js function to add the .days and .isInactive fields to each item
+	const sortedData = comparePurchaseUrgency(data);
+
```

Comment:
Since you're already sorting the data before passing it down to the `SubList` component, I would recommend also filtering the data at this point.